charts/consul/values.yaml

```diff
@@ -2185,6 +2185,11 @@ connectInject:
     # If this setting is false, you will need to install the Gateway API CRDs manually.
     manageExternalCRDs: true
 
+    # Enables Consul on Kubernets to manage only the custom CRDs used for Gateway API. If manageExternalCRDs is true 
+    # then all CRDs will be installed. If manageCustomerCRDs is true then only TCPRoute, gatewayclassconfigs and meshservice
+    # will be installed.
+    manageCustomCRDs: false
```

Comment:
If I'm understanding correctly, the combinations that matter are

```yaml
# manageCustomCRDs doesn't matter here and could be true or false, all CRDs will be installed
manageExternalCRDs: true
# manageCustomCRDs: false
```

```yaml
# This only installs TCPRoute, GatewayClassConfig and MeshService
manageExternalCRDs: false
manageCustomCRDs: true
```

```yaml
# No CRDs at all will be installed
manageExternalCRDs: false
manageCustomCRDs: false 
```

Reply:
That looks correct.